app/Http/Controllers/Backend/EventController.php

```diff
@@ -30,23 +30,21 @@ public static function suggestEvent(
                                                        'host'    => $host
                                                    ]);
 
-        if (config('app.telegram.token') !== null) {
```

Comment:
Why did you remove this catch?

Reply:
Wird im TelegramController sowieso nochmal gecheckt